candlepin.spec.tmpl

```diff
@@ -116,7 +116,7 @@ rm -rf %{buildroot}
 ## common
 %{__install} -d -m 755 %{buildroot}/%{_sysconfdir}/%{name}/certs/upstream/
 %{__install} -m 644 config/candlepin/candlepin-redhat-ca.crt %{buildroot}%{_sysconfdir}/%{name}/certs/upstream/
-%{__install} -m 644 config/candlepin/candlepin-redhat-r2-ca.crt %{buildroot}%{_sysconfdir}/%{name}/certs/upstream/
+%{__install} -m 644 config/candlepin/candlepin-authority.crt %{buildroot}%{_sysconfdir}/%{name}/certs/upstream/
```

Comment:
@nikosmoum I was uncertain on what the name of the new certificate should be? Should we establish a naming convention the certificates?

Reply:
Good question, but that brings something else to mind: If I remember correctly, the new CA cert should be able to validate signatures coming from both the old and new certs. So we can entirely replacing the old certificate with the new one (and keep the name the same). But that needs testing, which would be: Export manifest from Prod (using the old cert atm), and try to import to local candlepin that **only** has the new cert (no old, no r2). The expected outcome would be it works and we go forward with only the new cert. If it fails, then we need both certs and we'll think of a name for the new one then.

I'd also throw in a negative test just to be sure: Export manifest from Stage (using the new cert atm), and try to import to local candlepin _without_ this change. The expected outcome would be the import fails.